test_scripts/RC/CLIMATE_RADIO/OnRemoteControlSettings/008_Allowed_false.lua

```diff
@@ -12,9 +12,8 @@
 --
 -- SDL must:
 -- 1) store RC state allowed:false internally
--- 2) assign HMILevel none to all registered applications with appHMIType REMOTE_CONTROL and send OnHMIStatus (NONE) to such apps
--- 3) keep all applications with appHMIType REMOTE_CONTROL registered
--- 4) unsubscribe all REMOTE_CONTROL applications from OnInteriorVehicleData notifications for all HMI modules internally
+-- 2) keep all applications with appHMIType REMOTE_CONTROL registered
```

Comment:
Propose to update to "keep all applications with appHMIType REMOTE_CONTROL registered and in current HMI levels "